### PR TITLE
[PLAYER-4260] Create an activity that illustrates how you can set the

### PR DESCRIPTION
**Pull request by AlinaVoronkovaEpam**
initial playback speed using Options and set a playback speed at a player directly.

**Please merge it AFTER [PLAYER-3558] merging to Android SDK.**

### CODE REVIEW

**Review by SZadorozhnyiEpam — APPROVED** [review]

File: OoyalaSkinSampleApp/app/build.gradle
Comment (brol1dev) [inline OoyalaSkinSampleApp/app/build.gradle]: let's not add VR to the OoyalaSkinSampleApp, we don't want to add everything in this one. Also removed all the files related to VR in this app.

Instead add an activity in the VR sample app to show how to use rate on that one.

Thanks.

**Review by brol1dev — CHANGES_REQUESTED** [review]
Added one comment. Please update. Thanks

File: OoyalaSkinSampleApp/app/build.gradle
Comment (AlinaVoronkovaEpam) [reply OoyalaSkinSampleApp/app/build.gradle]: Ok. np. I'll do it tomorrow.

**Review by SZadorozhnyiEpam — APPROVED** [review]
LGTM

**Review by SergeyBicarte — APPROVED** [review]

**Comment by akshaydch11** [comment]
Lets wait for merge until native skin and sdk PRs are merged.

**Review by akshaydch11 — APPROVED** [review]

**Comment by brol1dev** [comment]
We don't really need this PR. The playback rate can be already set through the Skin SDK UI. There's no need to add separate use cases just for playback speed rate if this is applied for every single asset.

However we can add a use case in the AdvancedPlaybackSampleApp that will show how to set initial playback programatically.

I won't close yet to see if there's any code that we can use but this one will be rejected.

**Pull request closed without merging**
